packages.dhall

```diff
@@ -70,7 +71,7 @@ in  upstream
         , "unfoldable"
         ]
         "https://github.com/input-output-hk/purescript-marlowe.git"
-        "main"
+        "fb588282974a1aac67299b6ce37c818b156e28ab"
```

Comment:
`spago2nix` fails unless we provide gitrev hashes. I went in and manually copy-pasted the latest main commit sha.

Reply:
Thanks and sorry for this inconvenience! We should use revs and... `tag` all our own packages as well ;-)